blockchain/indexers/common.go

```diff
@@ -142,8 +142,7 @@ type Indexer interface {
 	// WaitForSync subscribes clients for the next index sync update.
 	WaitForSync() chan bool
 
-	// UpdateSyncSubscribers closes all sync subscriber update channels
-	// and purges the sync subscriber set.
+	// UpdateSyncSubscribers updates all indexer synchronization subscribers.
```

Comment:
This comment doesn't actually say anything.  I have no idea what its purpose is and under what circumstances I need to call it from reading this.

Reply:
Updated the function name and comment.